src/js/components/Form/Form.js

```diff
@@ -85,15 +85,21 @@ const Form = forwardRef(
       setValue(prevValue => {
         const nextValue = { ...prevValue };
         nextValue[name] = data;
-
         // re-run any validations, in case the validation
         // is checking across fields
         setErrors(prevErrors => {
           const nextErrors = { ...prevErrors };
           Object.keys(prevErrors).forEach(errName => {
             if (validations.current[errName]) {
               const nextError = validations.current[errName](data, nextValue);
-              updateErrors(nextErrors, errName, nextError);
+              if (!nextError && nextErrors[name]) {
+                updateErrors(nextErrors, name, nextErrors[name]);
```

Comment:
I think `nextErrors[name]` is already set before we make this call. Do we need this?